CHANGELOG.md

```diff
@@ -23,12 +23,40 @@ CHANGELOG
 
 Swift 5.0
 ---------
+
+* Swift 3 mode has been removed. Supported values for the `-swift-version`
+  flag are `4`, `4.2`, and `5`.
+
+* [SE-0228][]:
+
+  String interpolation has been overhauled to improve its performance,
+  clarity, and efficiency.
+
+  Note that the old `_ExpressibleByStringInterpolation` protocol has been
+  removed; any code making use of this protocol will need to be updated
+  for the new design.
```

Comment:
Should we mention `#if compiler` as a useful thing here? (That's how I tested an internal Apple project that was using `_ExpressibleByStringInterpolation`.)